GomokuClient/packages/gomoku-core/src/pages/JoinGame.tsx

```diff
@@ -142,9 +128,9 @@ const JoinGame = ({ gameHistory, playerID }: JoinGameProps) => {
                 {isMobile && (
                   <GameTimeMobile
                     {...commonGameTimeProps}
-                    player={players[1]}
+                    player={players.white}
                     //TODO: we should not pass both one of them should be required both not both at same time
-                    blackTimeLeftMobile={playerClock.blackTimeLeft}
+                    timeLeft={playerClock.blackTimeLeft}
```

Comment:
I still see on mobile NaN, have you checked?

<img width="620" alt="Screenshot 2024-11-06 at 19 56 43" src="https://github.com/user-attachments/assets/91f269c6-a02d-4781-b6b6-852cf0a2c0c5">

Also, if you already touched this logic 😅, maybe you can fix our issue with going negative, jsut by sending event from server or receiveing and also putting fallback for clock that it can't do minus value

<img width="425" alt="Screenshot 2024-11-06 at 19 58 24" src="https://github.com/user-attachments/assets/56b198c8-887d-4f3d-83db-35186f281c3c">

Reply:
![image](https://github.com/user-attachments/assets/eb1437d8-5115-4dae-ba74-29f8c21a7c26)

Reply:
Probably you are not on my branch or some cache stuff. I removed that "Anonymous" logic fallback completely

Reply:
For now handled negative time in client by showing 0 time if time is negative